clients/client/src/main/java/org/projectnessie/client/StreamingUtil.java

```diff
@@ -114,6 +115,34 @@ private StreamingUtil() {
         .generateStream(ref, maxRecords);
   }
 
+  /**
+   * Default implementation to return a stream of reflog entries, functionally equivalent to calling
+   * {@link NessieApiV1#getRefLog()} with manual paging.
+   *
+   * <p>The {@link Stream} returned by {@code getReflogStream(OptionalInt.empty())}, if not limited,
+   * returns all reflog entries.
+   *
+   * @param api The {@link NessieApiV1} to use
+   * @return stream of {@link RefLogResponse.RefLogResponseEntry} objects
+   */
+  public static Stream<RefLogResponse.RefLogResponseEntry> getReflogStream(
```

Comment:
No tests for this?

Reply:
the filter testcase is using this streamUtil itself. So, it is covered.